core/src/main/java/tachyon/worker/hierarchy/StorageDir.java

```diff
@@ -117,6 +125,7 @@ public void accessBlock(long blockId) {
     synchronized (mLastBlockAccessTimeMs) {
       if (containsBlock(blockId)) {
         mLastBlockAccessTimeMs.put(blockId, System.currentTimeMillis());
+        updateReferenceFrequency(blockId, -1);
```

Comment:
Rather than use -1 can you make this a static constant? That way the code is more clear what's going on